openshift_scalability/config/golang/nodeVertical-labeled-nodes.yaml

```diff
@@ -7,10 +7,10 @@ ClusterLoader:
       tuning: default
       ifexists: delete
       pods:
-        - num: 1000
-          image: gcr.io/google_containers/pause-amd64:3.0
-          basename: pausepods
-          file: pod-pause-labeled-nodes.json
+        - num: 660
+          image: openshift/hello-openshift
+          basename: hello
+          file: ../../content/pod-pause-labeled-nodes.json
```

Comment:
We want to use pause pods for this test because they are the smallest possible container to run that we have.   They make the test purely a test of the control plane and node infrastructure - no user or OS code involved.